src/net/sourceforge/kolmafia/request/SpleenItemRequest.java

```diff
@@ -51,27 +52,10 @@ public static final int maximumUses(
     int spleenLeft = limit - KoLCharacter.getSpleenUse();
     int usableMaximum = spleenHit == 0 ? Integer.MAX_VALUE : spleenLeft / spleenHit;
 
-    switch (itemId) {
-      case ItemPool.TURKEY_BLASTER -> {
-        UseItemRequest.limiter = "daily limit";
-        return Math.min(usableMaximum, (3 - Preferences.getInteger("_turkeyBlastersUsed")));
-      }
-      case ItemPool.MOJO_FILTER -> {
```

Comment:
this is not in dailylimits.txt

and also indicates you should be using the `isDaily` function on Preferences instead of checking a starting underscore

Reply:
Great point!

Reply:
Wait this shouldn't even be in this file

Reply:
Yeah this was just non-reachable code in reality.

Reply:
is it in `UseItemRequest`?

Reply:
it is.